src/widget/widgetOrchestrator.js

```diff
@@ -87,12 +87,12 @@ export default class WidgetOrchestrator {
    *
    * @returns {Element} the created button
    */
-  addOrbitButton(widgetElement, platform) {
-    if (!!widgetElement.querySelector(`obe-${platform}-button`)) return;
+  addOrbitButton(widgetElement, platform, page) {
+    const buttonElementName =
```

Comment:
? Is this needed?  Should the getButtonElementName should default to obe-plaform-button?

Reply:
Related to my comment above about “teaching” `Page` classes the platforms they’re on